hil/ext/switches/dell.py

```diff
@@ -29,6 +29,7 @@
 from os.path import dirname, join
 from hil.errors import BadArgumentError
 from hil.model import BigIntegerType
+from hil.ext.switches.switch_common import _should_save
```

Comment:
Two things:

1. The leading underscore convention means "this is an implementation detail," so you shouldn't name it that if it's meant to be exported from another module.
2. Instead, of `switches.switch_common`, maybe just call it `switches.common`.